Challenge.py

```diff
@@ -0,0 +1,12 @@
+from collections import Counter
+
+f = open("text.txt", "r")
```

Comment:
When opening file it is mandatory to close it when its no longer needed. you can use "close" method, but better implementation would be to use "with...open"